cli/install/viz.go

```diff
@@ -0,0 +1,67 @@
+package install
+
+// Viz defines the primary Conduit Grafana dashboard, installed via the `conduit install` command.
+const Viz = `{
+      "rows": [
```

Comment:
Is the indentation on this text blob required? If not, I'd prefer to keep it unindented, similar to to the yaml blob in template.go. I think that will help us keep the indentation properly set going forward.

```
const Viz = `{
  "rows": [
    {
      "collapse": false,
      "height": "50px",
      ...
    }
  ],
  "title": "conduit-viz"
}
```

Reply:
Because these strings are injected into `template.go` via Go templating, and need to be indented to conform to proper YAML, I can't find an easy way to not indent these strings.

One option would be to have the Go code read the `Viz` string and programmatically add indents.

I'd definitely like to optimize for easily editing dashboards and dropping them into these files. Ideas welcome.

Reply:
Ahhh, I totally missed that this was being substituted into yaml. I think it's fine to leave it as is in that case. I could also imagine the setup that you suggested with programmatically adding imports. It would look something like this: https://play.golang.org/p/0sMYo34qyQL . But I think both approaches are valid, so.... carry on :)

Reply:
I like that approach! I've updated #420 to call this out in a subsequent task.